src/main/java/uk/gov/hmcts/darts/common/entity/AnnotationEntity.java

```diff
@@ -48,4 +53,22 @@ public class AnnotationEntity extends CreatedModifiedBaseEntity {
 
     @Column(name = "is_deleted")
     private boolean deleted;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = AnnotationDocumentEntity_.ANNOTATION)
```

Comment:
i thought it was better to be eager:-
1. I think an annotationEntity on its own would be useless without the associated documents, i tried to think of when it would get used on its own, and couldn't think of a scenario.
2. We are only really expecting 1 document per annotation. the only reason its a OneToMany is for potential future proofing